src/commands/start/hashi_vault.py

```diff
@@ -0,0 +1,129 @@
+import asyncio
+import logging
+import sys
+
+import click
+from eth_typing import ChecksumAddress
+from web3.types import Gwei
+
+from src.commands.start.base import load_operator_config, start_base
+from src.commands.start.common_option import add_common_options, start_common_options
+from src.common.typings import ValidatorType
+from src.common.utils import log_verbose
+from src.config.settings import DEFAULT_HASHI_VAULT_PARALLELISM, settings
+
+logger = logging.getLogger(__name__)
+
+
+@click.option(
+    '--hashi-vault-url',
+    envvar='HASHI_VAULT_URL',
+    help='The base URL of the vault service, e.g. http://vault:8200.',
+)
+@click.option(
+    '--hashi-vault-token',
+    envvar='HASHI_VAULT_TOKEN',
+    help='Authentication token for accessing Hashi vault.',
+)
+@click.option(
+    '--hashi-vault-key-path',
+    envvar='HASHI_VAULT_KEY_PATH',
+    multiple=True,
+    help='Key path(s) in the K/V secret engine where validator signing keys are stored.',
+)
+@click.option(
+    '--hashi-vault-key-prefix',
+    envvar='HASHI_VAULT_KEY_PREFIX',
+    multiple=True,
+    help='Key prefix(es) in the K/V secret engine under which validator signing keys are stored.',
+)
+@click.option(
+    '--hashi-vault-parallelism',
+    envvar='HASHI_VAULT_PARALLELISM',
+    help='How much requests to K/V secrets engine to do in parallel.',
+    default=DEFAULT_HASHI_VAULT_PARALLELISM,
+)
+@click.option(
+    '--public-keys-file',
+    type=click.Path(exists=True, file_okay=True, dir_okay=False),
+    envvar='PUBLIC_KEYS_FILE',
+    help='Absolute path to the available validator public keys file. '
+    'Default is the file generated with "create-keys" command.',
+)
```

Comment:
Please check whether it's possible to read all the public keys from the hashi vault